modules/internal/DefaultAssociative.chpl

```diff
@@ -147,7 +147,51 @@ module DefaultAssociative {
         }
       }
     }
-  
+ 
+    iter these(param tag: iterKind) where tag == iterKind.standalone {
+      if debugDefaultAssoc then
+        writeln("*** In associative domain standalone iterator");
+      const numTasks = if dataParTasksPerLocale==0 then here.maxTaskPar
+                       else dataParTasksPerLocale;
+      const ignoreRunning = dataParIgnoreRunningTasks;
+      const minIndicesPerTask = dataParMinGranularity;
+      // We are simply slicing up the table here.  Trying to do something
+      //  more intelligent (like evenly dividing up the full slots, led
+      //  to poor speed ups.
+      // This requires that the zipppered domains match.
+      const numIndices = tableSize;
+      if debugAssocDataPar {
+        writeln("### numTasks = ", numTasks);
+        writeln("### ignoreRunning = ", ignoreRunning);
+        writeln("### minIndicesPerTask = ", minIndicesPerTask);
+      }
+
+      const numChunks = _computeNumChunks(numTasks, ignoreRunning,
```

Comment:
Basically is there a reason this is: 

```chapel
const numTasks = if dataParTasksPerLocale==0 then here.maxTaskPar
                 else dataParTasksPerLocale;
const ignoreRunning = dataParIgnoreRunningTasks;
const minIndicesPerTask = dataParMinGranularity;
const numIndices = tableSize;
if debugAssocDataPar {
  writeln("### numTasks = ", numTasks);
  writeln("### ignoreRunning = ", ignoreRunning);
  writeln("### minIndicesPerTask = ", minIndicesPerTask);
}

const numChunks = _computeNumChunks(numTasks, ignoreRunning,
                                    minIndicesPerTask,
                                    numIndices);

```

instead of: 

```chapel
const numIndices = tableSize;
const numChunks = _computeNumChunks(numIndices);
```